packages/node/src/sdk/index.ts

```diff
@@ -208,6 +208,9 @@ export function validateOpenTelemetrySetup(): void {
   }
 }
 
+const FALSY_ENV_VALUES = new Set(['0', 'false', 'no']);
+const TRUTHY_ENV_VALUES = new Set(['1', 'true', 'yes']);
```

Comment:
let's use the same values as we use [here](https://github.com/getsentry/sentry-javascript/blob/40a86d5c0b58839f40e6bacec98c491b5e0e63a2/packages/node/src/sdk/index.ts#L296) for consistency